i18n/fra/out/project/openProject.i18n.json

```diff
@@ -2,5 +2,6 @@
   "autolispext.project.openproject.onlyprjallowed": "Seuls les fichiers PRJ sont autorisés.",
   "autolispext.project.openproject.readfailed": "Impossible de lire le fichier de projet: ",
   "autolispext.project.openproject.malformedfile": "Fichier de projet incorrect: ",
-  "autolispext.project.openproject.label": "Ouvrir un projet"
+  "autolispext.project.openproject.label": "Ouvrir un projet",
+  "autolispext.project.openproject.nospaces": ""
```

Comment:
As much as you see a lot whole documents having spacing adjustments, you will also find a lot that don't and that is why I say this "fixed" the formatting my normalizing it.